atomicfu/build.gradle.kts

```diff
@@ -33,6 +59,12 @@ kotlin {
     wasmWasi {
         nodejs()
     }
+    
+    @Suppress("DEPRECATION") //https://github.com/Kotlin/kotlinx-atomicfu/issues/207
+    linuxArm32Hfp()
+
+    @OptIn(ExperimentalKotlinGradlePluginApi::class)
+    applyDefaultHierarchyTemplate()
```

Comment:
An approach like https://github.com/Kotlin/kotlinx-datetime/blob/7d2764a54011365b8c6749f3107fa628cd167abc/core/build.gradle.kts#L41-L85 can help avoid manually specifying `concurrentMain` and such.